app/containers/MessageBox/index.js

```diff
@@ -341,7 +341,7 @@ class MessageBox extends Component {
 
 	getPermalink = async(message) => {
 		try {
-			return await RocketChat.getPermalink(message);
+			return await RocketChat.getPermalink(message, 'message');
```

Comment:
same file line 615: `const permalink = await this.getPermalink(replyMessage);` - add second argument